backend/lib/config/gardener.js

```diff
@@ -88,11 +88,7 @@ module.exports = {
     this.assignEnvironmentVariables(config, env)
     const requiredConfigurationProperties = [
       'sessionSecret',
-      'apiServerUrl',
-      'oidc.issuer',
-      'oidc.client_id',
-      'oidc.client_secret',
-      'oidc.redirect_uri'
+      'apiServerUrl'
```

Comment:
> The oidc configuration validation should be skipped if the oidc configuration is completely empty.
please check if `config.oidc` is set. If yes, the oidc properties should be added to the required configuration properties

Reply:
It could look like this:
```js
const requiredConfigurationProperties = [
  'sessionSecret',
  'apiServerUrl'
]
if (config.oidc) {
  requiredConfigurationProperties.push(...[
      'oidc.issuer',	
      'oidc.client_id',	
      'oidc.client_secret',	
      'oidc.redirect_uri'
  ])
}
```

Reply:
@holgerkoser That’s almost exactly what I implemented. Aren’t the square brackets in `.push([])` in your example superfluous?